config.go

```diff
@@ -158,16 +157,16 @@ func (cfg *Config) CacheConfigLoad(path string) error {
 	return nil
 }
 
-// Configures whether generated wasm code will be interrupted via interrupt
+// SetInterruptable configures whether generated wasm code can be interrupted via interrupt
```

Comment:
Is this a Go idiom to start documentation with the name of the method? It seems slightly redundant to me because it's already attached to the method and otherwise feels repetitive

Reply:
https://golang.org/doc/effective_go.html#commentary